### PR TITLE
fix: renamed openSidebar to sidebarExpanded

### PR DESCRIPTION
**Pull request by rebelchris**
After discussing this with @sshanzel we decided to be more explicit about this setting.
What is does is either show the expanded sidebar or the minimised function.

Hence we renamed `openSidebar` to `sidebarExpanded`

### CODE REVIEW

File: src/schema/settings.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const typeDefs = /* GraphQL */ `
     """
     Whether to show the sidebar open
```
Comment (sshanzel) [inline src/schema/settings.ts:83]:
```suggestion
    Whether to show the sidebar in expanded form
```

File: src/schema/settings.ts [hunk 2]
```diff
@@ -134,7 +134,7 @@ export const typeDefs = /* GraphQL */ `
...
```
Comment (sshanzel) [inline src/schema/settings.ts:135]:
```suggestion
    Whether to show the sidebar in expanded form
```

**Review by sshanzel — APPROVED** [review]
Wow, thought you were just gonna apply an alias in the FE but you brought it to the next level 💯 

All looks good to me, just adjusted the definitions as well.

**Review by idoshamun — APPROVED** [review]
Make sure to resolve Lee's suggestion first but it looks good!

**Pull request merged**